mrdox.yml

```diff
@@ -0,0 +1,4 @@
+project-name: MrDox
+namespaces:
+  include:
+    - ::N
```

Comment:
Now this is interesting! There's missing newlines by the way. And if you are going to add this mrdox.yml please put it in its own folder.